eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -371,7 +371,7 @@ steps:
   inputs:
     targetPath: "$(Build.Repository.LocalPath)\\artifacts\\localizationArtifacts\\"
     artifactName: "localizationArtifacts"
-  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
+  condition: "and(eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
```

Comment:
It might be better to use `succeededorfailed()`, so that timed out or cancelled pipelines don't try to publish the artifacts.

Reply:
great point.